backend/src/packages/files/aws.service.ts

```diff
@@ -0,0 +1,86 @@
+import {
```

Comment:
I think it's better to store aws service (or not aws as @v0ldemar01 said) in libs/packages/aws folder, like jwt